weld_rt/cpp/Makefile

```diff
@@ -1,5 +1,5 @@
 OS = $(shell uname -s)
-LLVM_VERSION = 3.8
+LLVM_VERSION = $(shell llvm-config --version | cut -d . -f 1,2)
```

Comment:
This doesn't seem to work for me on MacOS with `brew install llvm`; the executables produced by this install don't have a `4.0` prefix.